end-to-end-untargeted-metabolomics.Rmd

```diff
@@ -794,9 +818,15 @@ legend("topright", col = col_phenotype,
        legend = names(col_phenotype), lty = 1, bty = "n")
 ```
 
-These EICs seems to indicate that our settings are suitable for this dataset.
-We can proceed to apply this algorithm to the entire dataset, and then extract
-the EICs for our two test ions to confirm that the process has been successful.
+Our custom settings seem thus to be suitable for our dataset. We now proceed and
+apply them to the entire dataset, extracting EICs again for the same ions to
+evaluate and confirm that chromatographic peak detection worked as
+expected. Note that we change the value for parameter `snthresh` to its default,
+because, as mentioned above, background noise estimation is more reliable when
+performed on the full data set. Parameter `chunkSize` of `findChromPeaks()`
```

Comment:
Should we relate it/mention to when parallel processing was set up at the beginning ? User might not make the link between that

Reply:
hm, difficult to say. I would like to keep this document as "general" as possible and keep the details maybe in the vignettes/tutorials? I'm just afraid we make this workflow too large.